### PR TITLE
Add container labels

### PR DESCRIPTION
**Pull request by fabricetriboix**
Also, reduce the number of layers by setting multiple environment variables at once

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
-ENV ACTIVEMQ_BASE $ACTIVEMQ_HOME
-ENV ACTIVEMQ_CONF /app/conf
-ENV ACTIVEMQ_DATA /app/data
-ENV ACTIVEMQ_TMP  /app/tmp
+# Environment variables: version
+ENV ACTIVEMQ_VERSION="5.16.2"
 
+# Environment variables: tarball stuff
+ENV ACTIVEMQ="apache-activemq-$ACTIVEMQ_VERSION" \
+    ACTIVEMQ="apache-activemq-$ACTIVEMQ_VERSION" \
```
Comment (JonHolman) [inline Dockerfile]: why is this one twice?

File: Dockerfile [hunk 2]
```diff
@@ -1,23 +1,28 @@
...
+        MAINTAINER="Armedia LLC"
 
-# Environment variables: ActiveMQ directories
-ENV ACTIVEMQ_HOME /app/activemq
-ENV ACTIVEMQ_BASE $ACTIVEMQ_HOME
-ENV ACTIVEMQ_CONF /app/conf
-ENV ACTIVEMQ_DATA /app/data
-ENV ACTIVEMQ_TMP  /app/tmp
+# Environment variables: version
+ENV ACTIVEMQ_VERSION="5.16.2"
```
Comment (JonHolman) [inline Dockerfile:10]: Can this be combined with the ENV block below?

File: Dockerfile (hunk 1 above)
Comment (fabricetriboix) [reply Dockerfile]: @JonHolman It's to make sure that it is set. Joking of course! I guess that's a copy/paste error, thanks for spotting that.

File: Dockerfile (hunk 2 above)
Comment (fabricetriboix) [reply Dockerfile:10]: @JonHolman No, because it is actually used in the next `ENV` block (I tried all in one block, but it doesn't work)

**Comment by naveen-armedia** [comment]
@fabricetriboix  are we going to run activemq and prometheus exporter jar on the same container..?

**Comment by fabricetriboix** [comment]
> @fabricetriboix are we going to run activemq and prometheus exporter jar on the same container..?

@naveen-armedia The Prometheus exporter is a Java agent, it doesn't run as a separate process.

**Review by JonHolman — APPROVED** [review]

**Review by naveen-armedia — APPROVED** [review]

**Pull request merged**